server/jetty-app/src/main/resources/logback.xml

```diff
@@ -19,12 +19,20 @@
     </encoder>
   </appender>
 
+  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+    <file>${deephaven.dataDir}/logs/myApp.log</file>
```

Comment:
Oops, meant to remove this. But it's a cool showing of what we can do w/ `deephaven.dataDir`

Reply:
Done